ibm_catalog.json

```diff
@@ -0,0 +1,103 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-icd-redis",
+      "label": "ICD Redis Module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "redis",
+        "icd"
+      ],
+      "short_description": "Implements an instance of IBM Cloud Databases for Redis.",
+      "long_description": "Use this module to provision and configure an IBM Cloud Database for Redis instance.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-redis/main/images/redis_icon.svg",
+      "features": [
+        {
+          "title": "Create Redis instance",
+          "description": "Create IBM Cloud Database for Redis instance."
+        },
+        {
+          "title": "Attach Users",
+          "description": "Attach list of users to the Redis instance."
+        },
+        {
+          "title": "Auto Scaling instance",
+          "description": "Rules to allow the Redis database instance to increase resources in response to usage."
+        },
+        {
+          "title": "Create Service Credentials Roles",
+          "description": "Create Service Credentials Roles for the ICD Redis database instance."
+        },
+        {
+          "title": "KMS encryption",
+          "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
+        },
+        {
+          "title": "CBR",
+          "description": "Create context-based restriction (CBR) rules for both instance and buckets."
+        }
```

Comment:
Suggest adding these features too:
- Redis configuration settings to tune your databases to your use-case
- Scaling Disk, Memory, and CPU (adjust the amount of resources available to your IBM Cloud® Databases for Redis deployment to suit your workload and the size of your data) **_[NOTE: This is manual scaling, not auto scaling which module also supports and you have included already]_**

Reply:
Consider putting auto-scaling and manual scaling together in one feature if it makes sense (e.g., `Supports both manual and auto-scaling to configure the Cloud Databases deployment to your needs.`)